client/webpack/dev.config.js

```diff
@@ -1,23 +0,0 @@
-var webpack = require('webpack');
-var base = require('./base.config');
-var config = base('development');
-
-
-// development overrides go here
-config.watch = true;
-
-// add poll-options for in vagrant development
-// See http://andrewhfarmer.com/webpack-watch-in-vagrant-docker/
-config.watchOptions = {
-  poll: 1000,
-  aggregateTimeout: 300,
-};
-
-// Set process.env.NODE_ENV to development to enable JS development aids.
-config.plugins.push(new webpack.DefinePlugin({
```

Comment:
What about these plugins?

Reply:
This is automatically defined in Webpack 4 based on the `mode`.